kubernetes/utils/create_from_yaml.py

```diff
@@ -60,46 +66,69 @@ def create_from_yaml(
         processing of the request.
         Valid values are: - All: all dry run stages will be processed
     """
+    if path.exists(yaml_file):
+        with open(path.abspath(yaml_file)) as f:
+            content = f.read()
+            try:
+                yaml_file = StringIO(content)
+            except TypeError:
+                yaml_file = StringIO(content.decode('utf-8'))
+
+    yml_document_all = yaml.safe_load_all(yaml_file)
+
+    for yml_document in yml_document_all:
+        create_from_dict(k8s_client, yml_document, verbose,
```

Comment:
as this function supports list objects this needs to catch the exception and build a new compound exception from it and raise that.

Reply:
This was the case before, I understood that this is not desired. I will revert this.

Reply:
This is already the case, `create_from_dict` gathers the exception, instead of throwing it immediately. When it is done, it throws the exception, which in turn will propagate to `create_from_yaml`.

Reply:
no create_from_dict gathers the exception from one object (which can be a list object)
create_from_yaml applies multiple objects which are all applied and only at the end an exception is 
the current code does not do that, it aborts after the first failure

as implemented you have to catch each exception and create a new one with all exceptions at the end (basically the same code as in from_dict but with the compound exception